serverless.yml

```diff
@@ -0,0 +1,22 @@
+service: lp-bot
+
+package:
+  exclude:
+    - node_modules/aws-sdk/**/*
+
+provider:
+  name: aws
+  runtime: nodejs12.x
+  stage: ${opt:stage, 'dev'}
+  region: ${opt:region, 'us-west-2'}
+  environment:
```

Comment:
The readme say to add 2 env vars to the function, but they are not defined here. Does that mean you are manually editing the deployed function in the console?

Reply:
Right now the answer is yes, but if there is a better way please let me know. I was thinking of doing param store but you also have to manually edit that on the console.

Reply:
You could create a config file to deploy those by stage.  Or you can create an ssm parameter and reference that via the code. 
It would be fine to create the parameter with serverless, but the value should be set manually.  But that can be done in the cli also so no need to actually log into the AWS console to make changes.